### ISSUE TITLE
Use HTTPS

### ISSUE THREAD

**Issue by andrewrabert** [Jomme5/AddressToGPSII#1]
Right now you're allowing user searches to be publically visible **and** you're leaking your own API key. Five minutes setting up mitmproxy got me this:
```
>> GET http://dev.virtualearth.net/REST%2Fv1%2FLocations?CountryRegion=EU&addressLine=new%20york&key=ApOcZ3jVnPdp5Ggrc6JemxY38iP1x44rB6yhJ2bcewMC4BvpFI87VMMqWOg3wpP3
       ← 200 application/json 467b 282ms
   GET http://dev.virtualearth.net/REST%2Fv1%2FLocations?query=new%20york&key=ApOcZ3jVnPdp5Ggrc6JemxY38iP1x44rB6yhJ2bcewMC4BvpFI87VMMqWOg3wpP3
       ← 200 application/json 1.22k 227ms
```
Use HTTPS.

**Comment by Jomme5** [comment]
I took the effort to make an encrypted store for the api-key to minimize chances getting it by string-dumping and then I simply forget the 's' in https. Working on it.  Key is burned, I suppose I'll have to change it. In the mean time, I have a version without 'api-key's' that does a simple bing-request with 'webscraping'. Don't know if it's legal though.

**Comment by andrewrabert** [comment]
Might also want to look into using [GPS Coordinates](https://www.gps-coordinates.net/api) as an alternative. Seems fairly open.

**Comment by Jomme5** [comment]
Solved in v1.3. Thanks for the contribution.
